script/test

```diff
@@ -1,5 +1,19 @@
-#!/bin/sh
+#!/bin/bash
+# See CONTRIBUTING.md for usage.
+
 set -ex
+
 docker build -t docker-compose .
```

Comment:
I would add a tag to each of these that is the commit sha so `docker-compose:$(git rev-parse --short HEAD)`